index.bs

```diff
@@ -5404,21 +5394,7 @@ partial dictionary MLOpSupportLimits {
             : {{MLInputOperandLayout/"nhwc"}}
             :: Let « |batches|, |inputHeight|, |inputWidth|, |channels| » be |inputShape|.
         </dl>
-    1. If |outputSizes| is given, then let « |outputHeight|, |outputWidth| » be |outputSizes|.
-    1. Otherwise:
-        1. Let |outputSizes| be the result of [=MLGraphBuilder/calculating conv2d output sizes=] given |inputHeight|, |inputWidth|, |windowDimensions|[0], |windowDimensions|[1], |padding|, |strides|, and |dilations|.
-        1. Let « |outputHeight|, |outputWidth| » be |outputSizes|.
-        1. Switch on |roundingType|:
-            <dl class=switch>
-                : {{MLRoundingType/"floor"}}
-                ::
-                    1. Set |outputWidth| to floor(|outputWidth|).
-                    1. Set |outputHeight| to floor(|outputHeight|).
-                : {{MLRoundingType/"ceil"}}
-                ::
-                    1. Set |outputWidth| to ceiling(|outputWidth|).
-                    1. Set |outputHeight| to ceiling(|outputHeight|).
-            </dl>
+    1. Let « |outputHeight|, |outputWidth| » be |outputSizes|.
```

Comment:
Should we add validation steps for user-supplied `outputSizes`? Chromium prototype does that: https://source.chromium.org/chromium/chromium/src/+/main:services/webnn/public/cpp/graph_validation_utils.cc;l=1179;drc=60130be50f68c89220d8ace62c87155025267a2c;bpv=1;bpt=1

If we add `outputSizes` validation steps, `windowDimensions`, `padding`, `strides` and `dilations` parameters would be required.